mlc_llm/core.py

```diff
@@ -126,6 +130,26 @@ class BuildArgs:
             "action": "store_true",
         },
     )
+    use_cutlass_attn: bool = field(
+        default=False,
+        metadata={
+            "help": (
+                "Offload attention operations to CUTLASS when the target is CUDA"
+                "and TVM has been built with CUTLASS enabled."
+            ),
+            "action": "store_true",
+        },
+    )
+    use_cutlass_norm: bool = field(
+        default=False,
+        metadata={
+            "help": (
+                "Offload layer and RMS norm operations to CUTLASS when the target is CUDA"
+                "and TVM has been built with CUTLASS enabled."
+            ),
+            "action": "store_true",
+        },
+    )
```

Comment:
can we merge them into one config: `use_cutlass="norm;attn"`?

Reply:
I want to stick to the existing convention to use boolean flags for this PR.

Reply:
Will be good future work to do